.github/workflows/emulator.yml

```
@@ -0,0 +1,25 @@
+name: Emulator-CI
+
+on:
+  push:
+    paths:
+      - 'emulator/**'
+
+jobs:
+  Emulator:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Clone Scalardl-tool repository
```

```suggestion
    - name: Clone scalardl-tools repository
```
BTW do we need this ? seems obvious.

---

Thank you review comment.
Workflow modified based on review comments.

---

I think `jobs.<job_id>.steps.name` is usefule since GitHub Action displays that step name.
We can immediately know which step fails.